### ISSUE TITLE
sirkiali misses tray icon, no way to reach settings after minimized startup

### ISSUE THREAD

**Issue by fago** [mhogomchungu/sirikali#74]
I'm using ubuntu 18.04 with gnome shell. While I've the AppIndicator-Support extensions https://extensions.gnome.org/extension/615/appindicator-support/ installed, there is still no tray icon for sirikali.

Unfortunately, I had to try to "minimize on startup" setting, now sirikiali is starting up minimized to tray *always* and I have no way to recover. I cannot see where sirikali stores its config either - so how can I revert this setting?

Unfortunately there is no startup option like "-e" for starting minimized that would force sirikali to startup with visible GUI.

**Comment by mhogomchungu** [comment]
What version of SiriKali are you using?

If its a recent version, you can set/unset to start minimized by setting the option at menu->settings->general options->start minimized.

I do not have Gnome installed making it impossible for me to see the application works on a Gnome desktop.

**Comment by fago** [comment]
sry - I'm using 1.3.3, from the deb 1.3.3-1.

Yeah, this setting exists as I did enable it. But the problem is I cannot reach the GUI any more now - sirikali always starts minimized and as I miss the tray icon I cannot make it show the GUI. Thus I'm unable to untick the setting.

Where are the user settings stored so I can reset it?

**Comment by 4O4** [comment]
On linux, the config should be in `$HOME/.config/SiriKali/SiriKali.conf`

**Comment by fago** [comment]
ha, I'm stupid - I was looking for "sirikali" (lowercase) and missed that there is an upper-case dir! As the binary is lower-case also I'd have expected that.

thanks a lot, I fixed it now.

Interestingly the tray icon is appearing now also, not sure what changed here. I just applied some general system-updates yesterday, maybe that helped. So it's seems like there is no bug in sirikali, thus closing this!

Thank you for your work also, this is very useful tool!
